.travis.yml

```diff
@@ -0,0 +1,18 @@
+language: objective-c
+
+notifications:
+  email:
+    on_success: never
+    on_failure: change
+
+env:
+  - APM_TEST_PACKAGES="linter"
```

Comment:
You should probably remove this line, `linter-flow` is supposed to automatically install this for the user so you might as well test that too :wink:.